### ISSUE TITLE
Map exported in Opendrive format, the curvature is not continuous

### ISSUE THREAD

**Issue by wisdom-bob** [lgsvl/simulator#1357]
hello, Here I hope to get your help.
Now, I find the map exported by LGSVL simulator, the curvature of the road planview is not continuous. if you resample the planview, you will find the angle of the point to point is not change smoothly. It sometimes will suddenly changed a lot.
This phenomenon is easy happened for  sharp turn. 
Is there any way to cover the problem?

**Comment by EricBoiseLGSVL** [comment]
Yes, we have some issues with our open drive exporter.  We are looking into it and hope to fix asap.

**Comment by lemketron** [comment]
(Tracking in AUTO-6869.)
